_targets.R

```diff
@@ -19,27 +22,174 @@ list(
 
   # Data targets
   tar_target(data_joined, bind_rows(data_raw), pattern = map(data_raw)),
-  tar_target(data_clean, clean_data(data_joined)),
+  tar_target(data_clean, clean_data(data_joined, region_lookup)),
   tar_target(data_holdout, make_data_holdout(data_clean)),
+
+  tar_target(participant_summary, make_participant_summary(data_clean)),
+  tar_target(region_lookup, make_region_lookup()),
+  tar_target(demog_table, make_demog_table(participant_summary)),
   tar_target(data_imp, make_data_imp(data_holdout, n_imps = 3)),
 
-  # Modelling targets
-  tar_target(model_list, make_model_list(data_imp)),
-  tar_target(model_diagnostics, make_model_diagnostics(model_list)),
+  #################################################################
+  ##                          MODELLING                          ##
+  #################################################################
+
+  tar_target(
+    model_list_by_age,
+    make_model_list(data_imp, moderator = "age", moderator_term = "11, 18, 35, 65")
+  ),
+  tar_target(
```

Comment:
Would it just be adding complexity if I changed this to dynamic branching instead? It would make the graph cleaner. But then also not that important if this would be easier to come back to?

Reply:
No I would prefer dynamic branching. I haven't done it before though so keen to see what you come up with.